service/history/ndc/workflow_resetter.go

```diff
@@ -894,6 +894,22 @@ func reapplyEvents(
 				return reappliedEvents, err
 			}
 			reappliedEvents = append(reappliedEvents, event)
+		case enumspb.EVENT_TYPE_WORKFLOW_EXECUTION_TERMINATED:
```

Comment:
I remember there are some other places actually. e.g. the ReapplyEvents() method in workflow context.

Reply:
updated